api.py

```diff
@@ -167,6 +167,44 @@ def list_group_members(team_id, group_id, **kwargs):
     return make_get_request('team/{}/group/{}/users'.format(team_id, group_id), **kwargs)
 
 
+def set_owner(team_id, resource_id, **kwargs):
```

Comment:
I think this should mention team in the name somewhere so it is not confused with transferring to a user, which will probably get a function call one day